src/utils/subscriberBreaches.test.ts

```diff
@@ -223,7 +272,51 @@ describe("getSubBreaches", () => {
       >
     ).mockResolvedValueOnce(breachesWithOneResolved);
 
-    const subBreaches = await getSubBreaches(subscriber, []);
+    const subBreaches = await getSubBreaches(subscriber, [], "nl");
+    expect(subBreaches.length).toEqual(1);
+    expect(subBreaches[0].isResolved).toBe(true);
+  });
+
+  it("returns that a breach containing a SSN is resolved for US users", async () => {
```

Comment:
Shouldn't we want to verify that a breach with an SSN is marked as _un_resolved for US users, but resolved for non-US users?

Reply:
Added a test case that also verify that breaches that include SSN are unresolved for US users in [b130578](https://github.com/mozilla/blurts-server/pull/4212/commits/b130578597fb532d57abedf23a2106d81eec726c).